src/renderer/Generics/redesign/MetricTypes/MetricTypes.tsx

```diff
@@ -133,9 +134,22 @@ export const MetricTypes = ({
         }
         labelText = 'Peers connected';
         break;
+      case 'memoryUsagePercent':
+        if (typeof statsValue === 'number') {
+          titleText = `${statsValue.toFixed(2)}%`;
+        } else {
+          titleText = `${statsValue}%`;
+        }
+        iconId = 'bolt';
```

Comment:
the icon appears white, but it should match the same color/shade as the other icons